bookstack/DOCS.md

```diff
@@ -127,9 +127,30 @@ MariaDB addon which should be installed prior to startup, this can be changed
 within the configuration to use an external MySql/MariaDB Database. Please note
 that there is no easy upgrade path between the two options.
 
+## SMTP Support
+
```

Comment:
Would be nice to split this into a separate PR and add some context to the chapter.

Reply:
okay will remove it and file a new PR if I have the time for it